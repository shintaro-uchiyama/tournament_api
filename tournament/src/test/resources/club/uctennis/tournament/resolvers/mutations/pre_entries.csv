"id", "tournament_id", "team_name", "representive_name", "email", "phone", "create_date", "update_date"