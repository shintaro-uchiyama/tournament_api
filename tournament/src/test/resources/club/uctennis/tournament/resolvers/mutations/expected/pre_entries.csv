"tournament_id", "team_name", "representive_name", "email", "phone", "update_date"
1, "team2", "representive2", "shintaro.0112@gmail.com", "phone2", null